apps/climatemapped-africa/README.md

```diff
@@ -0,0 +1,34 @@
+# hurumap
+
+hurumap helps journalists, researchers and activists transform their work with in-depth county-specific information. Get started now with datasets from Kenya.
```

Comment:
Probably needs updating since you've changed the name... and it's `HURUmap` not `hurumap`